CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ This is similar to [Angular's CHANGELOG.md](https://github.com/angular/angular/b
 ## [unreleased]
 
 ### Changed
+- Fix properties are now displayed in the same order as in the xml
```

Comment:
Please remove this line.

This is
a) not quite correct, because the issue is about the ordering defined at the type
b) not necessary as there was no release showing the properties at all.